### PR TITLE
Docker info binaries version update

### LINKED ISSUE #28282: do not display ContainerdCommitID in `info` on windows
related to https://github.com/docker/docker/pull/27955

/cc @jhowardmsft

### PR DESCRIPTION
**Pull request by mlaventure**
-  Fix version detection for docker-init
-  Remove extra binaries commit variables from windows build

Closes #28282

### CODE REVIEW

**Comment by mlaventure** [comment]
Just realize, this won't build in windows because the code still refers to the variables.

File: daemon/info_unix.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	v.ContainerdCommit.Expected = dockerversion.ContainerdCommitID
+	if sv, err := daemon.containerd.GetServerVersion(context.Background()); err == nil {
+		v.ContainerdCommit.ID = sv.Revision
+	} else {
+		logrus.Warnf("failed to retrieve containerd version: %v", err)
+		v.ContainerdCommit.ID = "N/A"
+	}
+
+	v.RuncCommit.Expected = dockerversion.RuncCommitID
+	if rv, err := exec.Command(DefaultRuntimeBinary, "--version").Output(); err == nil {
```
Comment (AkihiroSuda) [inline daemon/info_unix.go]: Should not we skip this on Solaris?

**Comment by jhowardmsft** [comment]
Nice! LGTM. Thanks :smile:

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Pull request merged**